CHANGELOG.md

```diff
@@ -5,8 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
-  - nothing yet
+## Unreleased [3.5.2]
+### Changed
+- Removed memory leaks that were identified by the code sanitizer
+
```

Comment:
This block is part of 3.5.1. Maybe we should have a 3.5.2 to go with 14.0.2, so the target merge branch should be dev/3.5.2. That'll make these 3.5.1 updates go away in the file changes for this PR.

Reply:
I've pushed a dev/3.5.2 branch (currently at same commit as 3.5.1) and have specified that as the target branch for the merge.